webapp/layouts/default.vue

```diff
@@ -206,6 +245,21 @@ export default {
     foldedup: function() {
       document.getElementById('nav-search-box').classList.remove('unfolded')
     },
+    toggleMobileMenuView() {
+      this.toggleMobileMenu = !this.toggleMobileMenu
+    },
+    redirectToRoot() {
+      this.$router.replace('/')
+      this.fetchPosts({ i18n: this.$i18n, filter: {} })
+    },
```

Comment:
this certainly feels like less than an ideal situation, and I'm open to other ideas how to clear the search queries for `Hashtags` and also refetch all the `Posts` to clear the search for `Hashtags/Categories`
@Tirokk @roschaefer @ogerly @ulfgebhardt 

it works with `location.reload()` as well, but I think that's worse.
it is not great user experience because it's not clear that anything is happening... maybe there could be a spinner/loader, but it's challenging because the logo link lives in `layouts/default.vue`, whereas the `Posts` list lives in `webapp/pages/index.vue`